docs/guide/learning/how-to-learn-vue.md

```diff
@@ -243,6 +243,68 @@ In this section, we will go over the most popular technologies and the struggles
 
 ### Laravel
 
+<!-- TODO: Missing Link -->
+You may be interested in [checking out the dev.to article](). It mirrors this section entirely (heading for heading), going into greater detail, while providing code samples and demos.
+
+Otherwise, read on for a brief overview of why Laravel and Vue is a fantastic combination.
+
+#### Reasons to use Laravel and Vue
+
+##### Scaffolded by default
```

Comment:
You can just use bullets here, instead of mini sections. The fact that these are in the article itself, is enough to give them more to read. Otherwise it gets pretty big too.

#### Pros
* Comes with Vue out of the box
* Has a simplified Webpack helper - [Laravel Mix] is a much simpler to configure API on top of Webpack
* Widely adopted - The Laravel creator and community have embraced Vue as it's default framework